standards/sql-standards.md

```diff
@@ -8,38 +8,308 @@ formatting issues and arguments in merge requests. The resulted in a Shed
 specific set of rules that plug and play nicely with the tool
 [SQLFLUFF](https://www.sqlfluff.com/).
 
-Add the configuration file called `sqlfluff.config` to your project directory or
+Add the configuration file called `.sqlfluff` to your project directory or
 somewhere referenced/linked to in your code editor of choice. The configuration
 setting:
 
 ```toml
 [sqlfluff]
 verbose = 1
 templater = jinja
-exclude_rules = L003,L036,L039
-output_line_length = 121
-sql_file_exts=.sql
+sql_file_exts = .sql
+dialect = tsql # See "Supported Dialects" for more info.
```

Comment:
I'd suggest setting this to a value that _won't_ work, otherwise someone's going to copy'n'paste this whole thing and wonder why everything's `tsql`-y.

Reply:
Good shout